apps/src/redux/runState.js

```diff
@@ -53,6 +56,12 @@ export default function reducer(state, action) {
     }
   }
 
+  if (action.type === SET_IS_DEBUGGING_SPRITES) {
+    return _.assign({}, state, {
+      isDebuggingSprites: state.isRunning ? action.isDebuggingSprites : state.isDebuggingSprites,
+    });
```

Comment:
This can be implemented with only one branch and no lodash dependency:
```js
if (action.type === SET_IS_DEBUGGING_SPRITES && state.isRunning) {
  return {...state, isDebuggingSprites: action.isDebuggingSprites};
}
```